### DISCUSSION TITLE
Code format standards

### DISCUSSION THREAD

**Discussion by rockerbacon** [rockerbacon/ice-cream-machine-core#12] [Technical]
We should define format standards for the source code of the main application, as well as for the source code of all auxiliary tools that may end up being created to support application development.

**Comment by rockerbacon** [comment]
I'd suggest we just roll with the standards from `go fmt` while the application is still in its experimental stage. We can revisit the standards as the project moves forward.

We're already building some Python tools in #10. I don't anticipate these being modified too often and they're unlikely to grow too much, so I'd suggest lean standards that we can follow using very simple automation.

**Reply by nlaranjeira01** [reply]
Agreed!

**Reply by rockerbacon** [reply]
What would you suggest for the Python standards?
